cli.js

```diff
@@ -170,6 +171,56 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
   }, async (argv) => {
     await handleCommand(argv, listHandler);
   })
+  .command('triggers', 'Manages triggers', yargs0 => yargs0
```

Comment:
I think an example section is needed here to show a very brief flow

Reply:
Added example for `upsert` command, `list` and `remove` are trivial
Not sure my example is good enough though...

Reply:
It's supposed to be `bn trigger`, singular: https://github.com/binaris/specs/blob/master/Redis%20Streams%20Trigger.md#trigger-configuration

Reply:
changed to `trigger`